apps/typescriptcourse/src/components/app/layout.tsx

```diff
@@ -6,7 +6,13 @@ import config from 'config'
 import isNull from 'lodash/isNull'
 
 export type LayoutProps = {
-  meta?: any
+  meta?: {
+    title?: string
+    description?: string
+    titleAppendSiteName?: boolean
+    url?: string
+    ogImage?: {url: string; alt?: string}
+  }
```

Comment:
typing this surface a few bugs where a more expanded meta prop was being passed in, it's noted below